src/parser.rs

```diff
@@ -113,14 +131,15 @@ impl MyAI {
                 }
             }
         }
-        self.my_board.board[y as usize][x as usize] = CellType::Enemy;
-        self.my_board.send_new_pos();
+        self.ai.receive_opponent_turn(&(y as u8, x as u8));
```

Comment:
I see that you're passing parameters as y first then x, which fixes a previous comment I have left, but I think we should rather pass parameters as x then y, and index as `[y][x]`

Reply:
Yep, changed it